storm/src/main/java/org/elasticsearch/storm/EsBolt.java

```diff
@@ -47,6 +47,7 @@
 import static org.elasticsearch.storm.cfg.StormConfigurationOptions.ES_STORM_BOLT_ACK;
 
 @SuppressWarnings({ "rawtypes", "unchecked" })
+@Deprecated
```

Comment:
Should we add a javadoc message here for the deprecation?

```java
/**
 * @deprecated Storm support is deprecated and will be removed in a future release
 */
```

Reply:
Yeah I had considered that, and then decided not to, and can't remember why. Just added it now though.